packages/multichain-account-service/src/providers/EvmAccountProvider.ts

```diff
@@ -260,6 +302,7 @@ export class EvmAccountProvider extends BaseBip44AccountProvider {
         );
```

Comment:
Just noticed we still had a `:getAccountByAddress` do we want to get rid of this one too?